OpenRA.Game/Sound/Sound.cs

```diff
@@ -11,17 +11,20 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
 using System.IO;
 using OpenRA.FileSystem;
 using OpenRA.GameRules;
 using OpenRA.Primitives;
 
 namespace OpenRA
 {
+#nullable enable
 	public interface ISoundLoader
 	{
-		bool TryParseSound(Stream stream, out ISoundFormat sound);
+		bool TryParseSound(Stream stream, [MaybeNullWhen(false)] out ISoundFormat sound);
```

Comment:
For the `Try` pattern, the `[MaybeNullWhen(false)]` attribute is quite handy. It allows us to say the result is not-null in the success case, but that if the method fails then `null` is possible.

This is better than `out Item? item` which would indicates it could be null even in the success case.